pkg/netpol/scan/scan_test.go

```diff
@@ -175,10 +183,3 @@ func TestNoK8sWorkloadResourcesFoundError(t *testing.T) {
 	noK8sWorkloadResourcesFound := &NoK8sWorkloadResourcesFoundError{}
 	require.True(t, errors.As(errs[0].Error(), &noK8sWorkloadResourcesFound))
 }
-
-func TestK8sWithOpenShiftResources(t *testing.T) {
-	dirPath := filepath.Join(testutils.GetTestsDir(), "acs_security_frontend_demos")
-	objs, errs := scanner.FilesToObjectsList(dirPath)
-	require.Empty(t, errs)
-	require.Len(t, objs, 7) // two deployments, 3 netpols, 2 services (skipping routes and configMaps)
-}
```

Comment:
I think we should keep this test, just try to find another openshift resources that is skipped, now that Route is not skipped.